### PR TITLE
adding Abhinav T B

### PR DESCRIPTION
**Pull request by abhinav-TB**
Fixes #

## Proposed Changes

-
-
-

### CODE REVIEW

File: src/components/json/worldData.js [hunk 1]
```diff
@@ -202,6 +203,22 @@ const data = [
...
+                "longitude":  76.299881,
+                "latitude": 9.981636 ,
+                "batch": 2020,
+                "photo": Abhinavtb,
+                "region": "Asia",
+                "skills": ["Web Dev", "AI/ML"],
+                "name": "Abhinav T B",
+                "bio": "Tech explorer | passionate developer | loves to automate things | problem solving | CSE undergrad @CUSAT ",
+                "url": "https://dsc.community.dev/cochin-university-of-science-and-technology/",
+                "chapterName": "cochin-university-of-science-and-technology",
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:217]:
```suggestion
                "chapterName": "Cochin University of Science and Technology",
```

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@abhinav-TB This PR is proper. Just one minor change. You can accept the suggestion and commit, after which your PR will be merged.

**Comment by abhinav-TB** [comment]
@aravindvnair99  done

**Review by aravindvnair99 — APPROVED** [review]

**Review by abhinavsri360 — APPROVED** [review]
Thanks for adding yourself!

**Pull request merged**